bio/varlociraptor/estimate-alignment-properties/wrapper.py

```
@@ -0,0 +1,12 @@
+import subprocess as sp
+
+log = snakemake.log_fmt_shell(stdout=False, stderr=True)
+
+sp.run(
+    "varlociraptor estimate alignment-properties "
+    f"{snakemake.params.get('extra', '')} {snakemake.input.ref} "
```

Ruff

Undefined name `snakemake`

(F821)